### ISSUE TITLE
Automated reporting of newly assembled genomes

### ISSUE THREAD

**Issue by batson** [czbiohub/sc2-illumina-pipeline#38]
When new genomes are assembled, we should generate a report.

This should include the number passing quality control and the number failing it, together with the sample names for all of those. We would also have the coverage maps for the successfully assembled genomes. To start with, this could just be an email or slack message with links to `filtered.stats.tsv` and the coverage plots folder. Later, this might be a more digestable report, which would allow one to QC the run at a glance.

Basically, @amykczb wants to be notified without having to ping @samlhao.

It's possible this could interface with the ___drumroll___ database.
